mne/tests/test_epochs.py

```diff
@@ -2450,6 +2450,44 @@ def test_events_list():
     assert_array_equal(epochs.events, np.array(events))
 
 
+def test_save_overwrite(tmpdir):
+    """Test saving with overwrite functionality."""
+
```

Comment:
This newline is a PEP8 error

https://travis-ci.org/mne-tools/mne-python/jobs/514830962#L3240

Reply:
@katarinaslama you should use a pep8 checker on your editor. Spyder, Sublime Text etc. all have options to enable it.

Reply:
Thanks! Changed my life =)